# Frame - Update User Frame
# -------------------------------
# Description:
#   Service-type for updating user frame
#   Used to update configuration of a defined custom user frame
#
# Version:
#   0.1 -   Initial version
#           [09.01.2024] - Jan T. Olsen
# -------------------------------

# Service-Request
# -------------------------------
# User-Frame name
string name

# Reference frame
# Frame for which user frame pose is given relative to                  
string ref_frame

# Pose-RPY
# User-friendly representation of pose
robot_toolbox/PoseRPY pose_rpy
---

# Service-Response
# -------------------------------
# Service result 
# Successful/Unsuccessful [true/false]
bool success       

# Service-Message              
string message